components/Cards/src/MDCCardCollectionCell.m

```diff
@@ -379,33 +376,32 @@ - (void)updateImageAlignment {
   switch (verticalImageAlignment) {
     case MDCCardCellVerticalImageAlignmentTop:
       yAlignment =
-          MDCCardCellSelectedImagePadding + CGRectGetHeight(self.selectedImageView.frame)/2;
+          MDCCardCellSelectedImagePadding + CGRectGetHeight(self.selectedImageView.frame) / 2;
       break;
     case MDCCardCellVerticalImageAlignmentCenter:
-      yAlignment = CGRectGetHeight(self.bounds)/2;
+      yAlignment = CGRectGetHeight(self.bounds) / 2;
       break;
     case MDCCardCellVerticalImageAlignmentBottom:
       yAlignment = CGRectGetHeight(self.bounds) - MDCCardCellSelectedImagePadding -
-          CGRectGetHeight(self.selectedImageView.frame)/2;
+                   CGRectGetHeight(self.selectedImageView.frame) / 2;
```

Comment:
It may be because it continues an arithmetic/logical expression from the previous line.

The Google C++ style guide has this styling, and I believe the Objective-C style guide originally inherited from that document.
https://google.github.io/styleguide/cppguide.html#Boolean_Expressions